enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/document/DocumentProcessingService.java

```diff
@@ -200,6 +202,7 @@ public void checkDocumentSubmitWithProvider(OwnerId ownerId, DocumentResultEntit
         }
 
         documentResultEntity.setErrorDetail(docSubmitResult.getErrorDetail());
```

Comment:
There seems to be a legacy mixture of error and reject reason (an old relic).

Let's consider something like
```
if (docSubmitResult.getErrorDetail() != null) {
  documentResultEntity.setErrorDetail(docSubmitResult.getErrorDetail());
  documentResultEntity.setErrorOrigin(ErrorOrigin.DOCUMENT_VERIFICATION);
}
```